src/olympia/reviewers/tests/test_views.py

```diff
@@ -5510,7 +5516,7 @@ def test_versions_that_are_flagged_by_scanners_are_highlighted(self):
                     results={'matchedRules': [customs_rule.name]},
                 )
 
-        with self.assertNumQueries(51):
+        with self.assertNumQueries(55):
```

Comment:
Yeah. The reason this one is increasing is that now there are some activity logs that weren't there before, that in turn load add-on, version and user. That's why the "main" test with query counts with the explanation hasn't changed (and was already higher than 51 because it already had a couple of activity logs for a different scenario).